### DISCUSSION TITLE
حل مشکل گوگل کپچا و گوگل 403

### DISCUSSION THREAD

**Discussion by ali-system** [Gozargah/Marzban#181] [Ideas]
با سلام 

`https://github.com/jinwyp/one_click_script
`
یک اسکریپت کاملا کارامد که ازش استفاده میکنیم

کافی کد 

`bash <(curl -Lso- https://git.io/kernel.sh)`

 در سرورتون وارد کنید تا پنل اسکریپت برای شما بیاد
بعد 

عدد 12

[https://github.com/jinwyp/one_click_script/blob/master/docs/readme3_en.png?raw=true](https://github.com/jinwyp/one_click_script/blob/master/docs/readme3_en.png)

برای ما اتومات وایرگاد و کلودفلر وارپ نصب میکنه 

وتنظیماتش فعال میکنه
بعد اون کافی y برای گزینه بعد انتخاب کند

در اخر هم عدد 1 برای انتخاب استفاده از ip ورژن 6

**Comment by 7follower** [comment]
ممنون ، اما کسی ورژن 6 نداشته باشه چطور میشه ؟ ؟

**Reply by alisystemit** [reply]
بله عزیزم

**Comment by abedin70** [comment]
عالی بود

**Reply by l3lack-eyes** [reply]
همه ترافیک از وارپ رد میشه ؟چطور کاستوم کنیم؟

**Comment by RPG-Dev420** [comment]
روی سرور نود هم باید نصب بشه ؟

**Comment by MirzaT1992** [comment]
کار نمیکند

**Comment by raminsh2001** [comment]
دمت گرم

**Comment by alisystemit** [comment]
مشکلی نیست شما بازم بزنید

**Comment by alisystemit** [comment]
هیچی از وارپ رد نمیشه
کلا فقد با وایرگاد ای پی v6 توی لوکال سرور میبنده ولی بازم ای پی قابل
استفاده هست

**Comment by alisystemit** [comment]
بله

**Comment by ali-system** [comment]
درست انجام ندادین
